rag/sql/security.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple
+import hashlib
+import os
+
+SENSITIVE_KEYWORDS = {"DELETE", "DROP", "UPDATE"}
```

Comment:
So, we will be able to change the database through the chatbot?

Reply:
You can if you have the signature added in the user's query. Otherwise, it will be rejected

Reply:
I think this could constitute as a new feature of changing the database by querying to the chatbot. Quite interesting! Though, personally, it's hard for me to trust the chatbot at all for any database update.

Otherwise, if this won't be a new feature, I think simply blocking any query that will change the database without signature checking should be enough.